deployment/enterprise_sso/enterprise_aws_sso_stack.py

```diff
@@ -247,49 +252,57 @@ def __init__(self, scope: Construct, construct_id: str, **kwargs) -> None:
             ],
         )
 
+        common_layer_path = Path("src/layers/common")
```

Comment:
Would it make sense to have all the path variables in one place, so that you don't need to search for them?

Reply:
sure, will move.